scripts/build

```diff
@@ -8,5 +8,6 @@ if [ -d "src/" ]; then
 	#cp -a -R src/* lib/
 fi
 cp test/cucumber/features/node_js/Readme.md ./
-cp src/meta-schema.json lib/meta-schema.json
+cp src/meta-schema-v3.json lib/meta-schema-v3.json
+cp src/meta-schema-v4.json lib/meta-schema-v4.json
```

Comment:
I'd prefer to use a `.coffee` file extension so the build script doesn't need to handle it by itself (coffee compile will handle it when building files). But YMMV.